### PR TITLE
docs: generate swagger for the new global fee query

### PR DESCRIPTION
**Pull request by sainoe**
## Description

Update the swagger query definition using the refactored Global Fee protos.

Closes: #XXXX

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] included the correct `docs:` prefix in the PR title
- [x] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] Confirmed the correct `docs:` prefix in the PR title
- [ ] Confirmed all author checklist items have been addressed 
- [ ] Confirmed that this PR only changes documentation
- [ ] Reviewed content for consistency
- [ ] Reviewed content for thoroughness
- [ ] Reviewed content for spelling and grammar
- [ ] Tested instructions (if applicable)

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -2725,6 +2725,7 @@ github.com/stretchr/testify v1.7.5/go.mod h1:yNjHg4UonilssWZ8iaSj1OCr/vHnekPRkoO
 github.com/stretchr/testify v1.8.0/go.mod h1:yNjHg4UonilssWZ8iaSj1OCr/vHnekPRkoO+kdMU+MU=
 github.com/stretchr/testify v1.8.1/go.mod h1:w2LPCIKwWwSfY2zedu0+kehJoqGctiVI29o6fzry7u4=
 github.com/stretchr/testify v1.8.2/go.mod h1:w2LPCIKwWwSfY2zedu0+kehJoqGctiVI29o6fzry7u4=
+github.com/stretchr/testify v1.8.3/go.mod h1:sz/lmYIOXD/1dqDmKjjqLyZ2RngseejIcXlSw2iwfAo=
```
Comment (mpoke) [inline go.sum]: A docs PR should not update go.sum

**Review by mpoke — CHANGES_REQUESTED** [review]

**Review by mpoke — APPROVED** [review]

File: docs/client/gaia/globalfee/v1beta1/query.swagger.json [hunk 2]
```diff
@@ -47,14 +47,36 @@
...
+          },
+          "description": "bypass_min_fee_msg_types defines a list of message type urls\nthat are free of fee charge."
+        },
+        "max_total_bypass_min_fee_msg_gas_usage": {
+          "type": "string",
+          "format": "uint64",
+          "description": "max_total_bypass_min_fee_msg_gas_usage defines the total maximum gas usage\nallowed for a transaction containing only messages of types in bypass_min_fee_msg_types\nto bypass fee charge."
+        }
+      },
+      "description": "Params defines the set of module parameters."
```
Comment (mmulji-ic) [inline docs/client/gaia/globalfee/v1beta1/query.swagger.json:73]: This should be a bit more descriptive.

File: docs/client/gaia/globalfee/v1beta1/query.swagger.json [hunk 3]
```diff
@@ -47,14 +47,36 @@
...
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "bypass_min_fee_msg_types defines a list of message type urls\nthat are free of fee charge."
+        },
+        "max_total_bypass_min_fee_msg_gas_usage": {
+          "type": "string",
+          "format": "uint64",
+          "description": "max_total_bypass_min_fee_msg_gas_usage defines the total maximum gas usage\nallowed for a transaction containing only messages of types in bypass_min_fee_msg_types\nto bypass fee charge."
```
Comment (mmulji-ic) [inline docs/client/gaia/globalfee/v1beta1/query.swagger.json:70]: That is a mouthful max_total_bypass_min_fee_msg_gas_usage, did you guys not have other options?

Comment (mmulji-ic) [reply docs/client/gaia/globalfee/v1beta1/query.swagger.json:70]: 🆗 this is not changeable now.

**Review by mmulji-ic — APPROVED** [review]

**Pull request merged**